pkg/api/server.go

```diff
@@ -12,14 +12,21 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
 package api
 
 import (
+	"context"
+	"crypto/tls"
 	"net"
 	"net/http"
 	"net/http/pprof"
+	"time"
+
+	"github.com/apache/apisix-ingress-controller/pkg/apisix"
```

Comment:
Sort the import.

Reply:
`make lint` should identify this, but it didn't